src/main/java/build/buildfarm/instance/AbstractServerInstance.java

```diff
@@ -1639,5 +1640,10 @@ public ServerCapabilities getCapabilities() {
         .build();
   }
 
+  @Override
+  public WorkerProfileMessage getWorkerProfile() {
+    return null;
```

Comment:
I'd make this throw an exception rather than return null here. SRE or other runtime